### DISCUSSION TITLE
Website side navigation has no `collapse-below` option

### DISCUSSION THREAD

**Discussion by declann** [quarto-dev/quarto-cli#6443] [Feature Requests]
### Description

From Quarto docs (and from experimentation), navbars have a handy `collapse-below` but sidebars don't.

https://quarto.org/docs/websites/website-navigation.html

On some of my pages I want a sidebar but it isn't critical; I tried to collapse it by default on most screensizes by increasing the responsive breakpoint via `collapse-below` but this is only possible for navbars, not sidebars.

I think would be useful (and more consistent) if sidebars also supported `collapse-below`? (especially if it happens to be easy?!)

**Comment by mcanouil** [comment]
I opened an issue to track this which I think is a good idea.

- #6457
